nbs/tsdataset.ipynb

```diff
@@ -54,7 +54,7 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.                df = pd.read_parquet(self.files_ds.files[idx])
</pre><p>I think we could have the convention of saving the files sorted by time, that way we can just read the temporal cols and assign them directly to data, e.g. <code>data = pd.read_parquet(self.files_ds.files[idx], columns=self.files_ds.temporal_cols).to_numpy()</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>